### ISSUE TITLE
Update BotKube with a skeleton for the new configuration syntax

### ISSUE THREAD

**Issue by mszostok** [kubeshop/botkube#656]
## Overview

Introduce only the syntax changes (nest under proper properties etc.). Support only one item on the list (hard-code `.[0].channel` etc.). 

## Acceptance Criteria

- BotKube works with a new syntax as described in the [binding configuration proposal](https://github.com/kubeshop/botkube/blob/main/docs/proposal/2022-06-14-configuration-bindings.md)
- Stretch: introduce the binding syntax.

Out of scope:
- docs changes
- `recommendation` property

## Reason

This will unblock the rest of the work in https://github.com/kubeshop/botkube/issues/596
